aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml

```diff
@@ -187,6 +267,14 @@
             <labelName>Nombre d'actualité</labelName>
             <language>fr</language>
         </label>
+        <label>
+            <labelName>How many news ?</labelName>
```

Comment:
In English: `Number of news` should be the more appropriate translation to `Nombre d'actualités`
Otherwise, in English, no non-breaking space between the sentence and (interrogation / exclamation / ... ) marks